src/tests/ftest/util/server_utils_params.py

```diff
@@ -112,6 +113,9 @@ def __init__(self, filename, common_yaml):
         self.control_log_file = LogParameter(log_dir, None, "daos_control.log")
         self.helper_log_file = LogParameter(log_dir, None, "daos_admin.log")
         self.telemetry_port = BasicParameter(None, 9191)
+        default_disable_vmd_val = os.environ.get("DAOS_DISABLE_VMD", "True")
+        default_disable_vmd = ast.literal_eval(default_disable_vmd_val)
```

Comment:
minor: On the one hand, using the ast module is kind of clever, but on the other hand, it's not so user-friendly. As it's written, the user has to use the strings "True" or "False" with that exact case in order for this to work. But the user could also specify strange things that evaluate properly (e.g. "0" or "1" or "42") and then potentially get undefined behavior. As this is only test code I wouldn't block on it, but I think this should probably be fixed in a follow-up so that it doesn't set the wrong precedent.

Reply:
Yes, Let me think about it... I will create a ticket to address this issue in a future PR.